test/int/framework/frameworkTestSupport.ts

```diff
@@ -44,12 +44,12 @@ export class TestProjectSpec {
         this._props = props;
         this._props.projectSrc = props.projectSrc || path.join(props.projectRoot, 'src');
         this._props.webRoot = props.webRoot || props.projectRoot;
-        this._props.outFiles = props.outFiles || path.join(props.projectRoot, 'out');
+        this._props.outDir = props.outDir || path.join(props.projectRoot, 'out');
         this._props.port = props.port || 7890;
         this._props.url = props.url || `http://localhost:${props.port}/`;
         this._props.launchConfig = props.launchConfig || {
             url: props.url,
-            outFiles: props.outFiles,
+            outDir: props.outDir,
```

Comment:
`outDir` is deprecated, you should stick with outFiles.

Reply:
Whoops. Reverted.